service/src/test/java/greencity/ModelUtils.java

```diff
@@ -172,11 +172,11 @@ public static HabitStatus getHabitStatus() {
 
     public static UserGoal getCustomUserGoal() {
         return UserGoal.builder()
-            .id(1L)
-            .user(User.builder().id(1L).email(TestConst.EMAIL).name(TestConst.NAME).role(ROLE.ROLE_USER).build())
-            .status(GoalStatus.DONE)
-            .customGoal(CustomGoal.builder().id(8L).text("Buy electric car").build())
-            .build();
+                .id(1L)
+                .user(User.builder().id(1L).email(TestConst.EMAIL).name(TestConst.NAME).role(ROLE.ROLE_USER).build())
+                .status(GoalStatus.DONE)
+                //.customGoal(CustomGoal.builder().id(8L).text("Buy electric car").build())
```

Comment:
Don't comment, delete if it is not needed

Reply:
done